### DISCUSSION TITLE
Dear author, would you consider checking the parameters that can directly transfer files by default?

### DISCUSSION THREAD

**Discussion by youhers** [rustdesk/rustdesk#2890] [General]
Because if you don’t pay attention when transferring files remotely, you need to copy the file twice each time, which is a bit cumbersome, or put this parameter in the configuration file and let the user choose whether it needs to be checked by default.

**Comment by rustdesk** [comment]
Sorry, not get what you are saying, :(

**Reply by youhers** [reply]
It is to copy and paste directly after connecting to the remote desktop. For now, it needs to be checked and allowed. This step is a little troublesome. In addition, during the pasting process, such as large file pasting, the desktop will not be controlled, and the operation can only be performed after the pasting is completed

**Comment by wxlg1117** [comment]
I think he is talking about showing a prompt when a target file exists and letting the user decide whether to overwrite or stop the transfer

The current setting of rustdesk is to overwrite if the target file exists when the file is transferred.

**Comment by zhangbo8418** [comment]
> Sorry, not get what you are saying, :(

Make a file search bar, it's too difficult to choose one file out of thousands of files to send. . .

**Comment by rustdesk** [comment]
flutter version will have this search bar @Kingtous

**Comment by grummbeer** [comment]
@youhers  can be closed?
